14-minimal/README.md

```diff
@@ -0,0 +1,250 @@
+NodeJS 14 container image
+=========================
+
+This container image includes Node.JS 14 as a [S2I](https://github.com/openshift/source-to-image) base image for your Node.JS 14 applications.
+Users can choose between RHEL, CentOS and Fedora based images.
+The RHEL images are available in the [Red Hat Container Catalog](https://access.redhat.com/containers/),
+the CentOS images are available on [Quay.io](https://quay.io/organization/centos7),
+and the Fedora images are available in [Fedora Registry](https://registry.fedoraproject.org/).
+The resulting image can be run using [podman](https://github.com/containers/libpod).
+
+Note: while the examples in this README are calling `podman`, you can replace any such calls by `docker` with the same arguments
+
+Description
+-----------
+
+Node.js 14 available as container is a base platform for 
+building and running various Node.js 14 applications and frameworks. 
+Node.js is a platform built on Chrome's JavaScript runtime for easily building 
+fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model 
+that makes it lightweight and efficient, perfect for data-intensive real-time applications 
+that run across distributed devices.
+
+Usage in OpenShift
+------------------
+In this example, we will assume that you are using the `ubi8/nodejs-14` image, available via `nodejs:14` imagestream tag in Openshift.
```

Comment:
This is not true. It will be something like `ubi8/nodejs-14-minimal` or similar, right?

Reply:
Indeed, usage for Openshift was rewritten completely.